tests/visualization_tests/test_edf.py

```diff
@@ -28,27 +40,36 @@ def test_plot_optimization_history(direction: str) -> None:
     study0 = create_study(direction=direction)
     study0.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf(study0)
+    _validate_edf_values(figure.data[0]["y"])
     assert len(figure.data) == 1
     assert figure.layout.xaxis.title.text == "Objective Value"
 
     # Test with two studies.
     study1 = create_study(direction=direction)
     study1.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf([study0, study1])
+    for points in figure.data:
+        _validate_edf_values(points["y"])
     assert len(figure.data) == 2
     figure = plot_edf((study0, study1))
+    for points in figure.data:
+        _validate_edf_values(points["y"])
     assert len(figure.data) == 2
 
     # Test with a customized target value.
     study0 = create_study(direction=direction)
     study0.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     with pytest.warns(UserWarning):
         figure = plot_edf(study0, target=lambda t: t.params["x"])
-    assert len(figure.data) == 1
+    data = figure.data
+    _validate_edf_values(data[0]["y"])
+    assert len(data) == 1
 
     # Test with a customized target name.
     study0 = create_study(direction=direction)
     study0.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf(study0, target_name="Target Name")
-    assert len(figure.data) == 1
+    data = figure.data
+    _validate_edf_values(data[0]["y"])
+    assert len(data) == 1
```

Comment:
For consistency as above.
```suggestion   
     _validate_edf_values(figure.data[0]["y"])
    assert len(figure.data) == 1
```